adopters.md

```diff
@@ -11,6 +11,7 @@ This is a list of TiDB adopters in various industries.
 
 | Company | Industry | Case studies |
 | :--- | :--- | :--- |
+| [Bank of China](https://en.wikipedia.org/wiki/Bank_of_China) | Banking | [English](https://pingcap.com/case-studies/how-bank-of-china-uses-a-scale-out-database-to-support-zabbix-monitoring-at-scale) |
```

Comment:
Please add this near Bank of Beijing and add its Chinese case study link. Thanks.